estuary/utils/story.py

```diff
@@ -276,6 +278,42 @@ def _get_plural_label(label):
             raise RuntimeError('{0} does not have a {1} relationship'.format(
                 story_node_display_name, rel_direction))
 
+    def get_recent_nodes(self):
```

Comment:
This doesn't belong in this class since it doesn't use any of the class variables or methods. It is also not part of constructing a story. It is a separate feature.

Reply:
where should I put it?

Reply:
You could create a new file in `estuary/utils`.